nodetypes/radon.nodes.VM/OpenStack/files/create/create.yml

```diff
@@ -20,6 +20,7 @@
         data:
           private_address: "{{ server_info.server.private_v4 }}"
           public_address: "{{ server_info.server.public_v4 }}"
+          public_ip:      "{{ server_info.server.public_v4 }}" 
```

Comment:
Is there a specific reason for defining a second variable for the same value (server_info.server.public_v4) ?

Reply:
ok, thanks for the clarification